Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.c

```diff
@@ -43,6 +43,13 @@
 #include "pas_random.h"
 #include <stdint.h>
 
+/* PlayStation does not current support the backtrace API. */
+#if !PAS_PLATFORM(PLAYSTATION)
+#include <execinfo.h>
+#else
+size_t backtrace(void **, size_t) { return 0; }
+#endif
```

Comment:
This is fine as-is, but a slightly better way to do this (in the future) would be to define a `PAS_HAVE(BACKTRACE)` or `PAS_HAVE(EXECINFO)` macro, and define that based on the platform.

No requirement to change this before landing, but having a more descriptive macro means you can remove the comment here and again in `PGMTests.cpp`.  :)